spec/spec_helper.rb

```diff
@@ -1,2 +1,37 @@
-$LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
-require 'plugin'
+require "codeclimate-test-reporter"
+require "simplecov"
+
+CodeClimate::TestReporter.start
+
+SimpleCov.start do
+  SimpleCov.maximum_coverage_drop 0.2
+  SimpleCov.start do
+    add_group  "Resources", "lib/cobrato/resources"
+    add_group  "Entities",  "lib/emites/entities"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.